### PR TITLE
[risk=low][no ticket] Remove a few UI packages we're no longer using

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -24,7 +24,6 @@
   "private": true,
   "dependencies": {
     "@clr/icons": "^0.11.12",
-    "@clr/ui": "^0.11.12",
```
Comment (jmthibault79) [inline ui/package.json:27]: Looks like we can now remove our dependency on Clarity UI, thanks to all the work @evrii did!  (please confirm)

Comment (jmthibault79) [reply ui/package.json:27]: We still have many Clarity Icons, so `@clr/icons` stays until https://precisionmedicineinitiative.atlassian.net/browse/RW-7683 is done

File: ui/package.json [hunk 2]
```diff
@@ -52,7 +51,6 @@
     "js-cookie": "^2.2.0",
     "lodash": "^4.17.21",
     "moment": "^2.29.4",
-    "ngx-popper": "^5.1.7",
```
Comment (jmthibault79) [inline ui/package.json:55]: also Angular

File: ui/package.json [hunk 3]
```diff
@@ -96,7 +94,6 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-simple-import-sort": "^8.0.0",
     "eslint-plugin-sort-keys-fix": "^1.1.2",
-    "lolex": "^3.0.0",
```
Comment (jmthibault79) [inline ui/package.json:99]: This is now FakeTimers, which is pulled in transitively by various Jest packages.  No need to list it here.

File: ui/package.json (hunk 1 above)
Comment (evrii) [reply ui/package.json:27]: Oh yeah, good call. Thanks for taking care of that. It seems to work just fine without it for me.

**Review by evrii — APPROVED** [review]
Looks like good cleanup to me. Tested by running locally and clicking around the application. Everything seemed to working as expected.

**Pull request merged**